django-backend/fecfiler/authentication/views.py

```diff
@@ -0,0 +1,38 @@
+from rest_framework import filters
+from rest_framework.viewsets import GenericViewSet
+from rest_framework.mixins import ListModelMixin
+from django.db.models import Value, CharField
+from django.db.models.functions import Concat
+from .models import Account
+from .serializers import AccountSerializer
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class AccountViewSet(GenericViewSet, ListModelMixin):
+    """
+        The Account ViewSet allows the user to retrieve the users in the same committee
+
+        The CommitteeOwnedViewset could not be inherited due to the different structure
+        of a user object versus other objects.
+            (IE - having a "cmtee_id" field instead of "committee_id")
+    """
+    queryset = Account.objects.annotate(
```

Comment:
Since you are defining get_queryset(), I would move this part of the queryset definition into the get_queryset().